### PR TITLE
Documentation Standard Updates

### PR DESCRIPTION
**Pull request by fivetran-joemarkiewicz**
This pull request was created automatically 🎉:

- [x] Ensure `.github/` files have been properly updated (templates are updated and auto releaser is included)
- [x] Confirm all appropriate end models are included in the `.quickstart/quickstart.yml` file as `public_models` (only for transform repos)
- Verify the README has made all necessary updates:
    - [x] Removal of emojis
    - [x] Replacing exclamation points with periods in sentences
    - [x] Adjusting all headers to ensure only one H1 heading
    - [x] Confirming a section header is included for Quickstart UI rendering (only for transform repos)
    - [x] Renaming `**model` to `**table**` (only for transform repos)
    - [x] Removal of Fivetran community and office hour resources (if necessary)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,30 +11,30 @@
...
 # Tiktok Ads Source dbt Package ([Docs](https://fivetran.github.io/dbt_tiktok_ads_source/))
-# 📣 What does this dbt package do?
+## What does this dbt package do?
 - Materializes [Tiktok Ads staging tables](https://fivetran.github.io/dbt_tiktok_ads_source/#!/overview/tiktok-ads-source/models/?g_v=1) which leverage data in the format described by [this ERD](https://fivetran.com/docs/applications/tiktok-ads/#schemainformation). These staging tables clean, test, and prepare your Tiktok Ads data from [Fivetran's connector](https://fivetran.com/docs/applications/tiktok-ads) for analysis by doing the following:
-  - Name columns for consistency across all packages and for easier analysis
-  - Adds freshness tests to source data
-  - Adds column-level testing where applicable. For example, all primary keys are tested for uniqueness and non-null values.
+- Name columns for consistency across all packages and for easier analysis
+- Adds freshness tests to source data
+- Adds column-level testing where applicable. For example, all primary keys are tested for uniqueness and non-null values.
```
Comment (fivetran-joemarkiewicz) [inline README.md:18]:
```suggestion
  - Name columns for consistency across all packages and for easier analysis
  - Adds freshness tests to source data
  - Adds column-level testing where applicable. For example, all primary keys are tested for uniqueness and non-null values.
```

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+contact_links:
```
Comment (fivetran-jamie) [inline .github/ISSUE_TEMPLATE/config.yml:1]: @fivetran-joemarkiewicz I noticed that there is already a `config.yml` file, but not in the `ISSUE_TEMPLATE` subdirectory - https://github.com/fivetran/dbt_tiktok_ads_source/blob/MagicBot/documentation-updates/.github/config.yml

Should we remove the other one?

Comment (fivetran-joemarkiewicz) [reply .github/ISSUE_TEMPLATE/config.yml:1]: Good catch! Let's remove the one that is not in the ISSUE_TEMPLATE subdirectory.

**Review by fivetran-jamie — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -103,15 +103,15 @@ vars:
     tiktok_ads_<default_source_table_name>_identifier: your_table_name 
 ```
 
-## (Optional) Step 5: Orchestrate your models with Fivetran Transformations for dbt Core™
+### (Optional) Step 5: Orchestrate your models with Fivetran Transformations for dbt Core™
 <details><summary>Expand for more details</summary>
```
Comment (fivetran-jamie) [inline README.md:107]:
```suggestion
<details><summary>Expand for more details</summary>
<br>
```

**Comment by fivetran-catfritz** [comment]
@fivetran-jamie @fivetran-joemarkiewicz  this might be a good time to remove the [2nd reviewer workflow ](https://github.com/fivetran/dbt_tiktok_ads_source/blob/main/.github/workflows/add-2nd-reviewer.yml).

**Pull request merged**